python/llm/setup.py

```diff
@@ -50,8 +50,13 @@
 llm_home = os.path.join(os.path.dirname(os.path.abspath(__file__)), "src")
 github_artifact_dir = os.path.join(llm_home, '../llm-binary')
 libs_dir = os.path.join(llm_home, "ipex_llm", "libs")
+
+cpu_torch_version_linux = 'torch == 2.1.2+cpu'
+cpu_torch_version_win = 'torch == 2.1.2'
+global cpu_torch_version
+cpu_torch_version = cpu_torch_version_linux
```

Comment:
Should we use
["torch==2.1.2+cpu;platform_system=='Linux'", "torch==2.1.2;platform_system=='Windows'" instead]